src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -130,4 +130,8 @@ public static final class prefShooter {
         0);
 
   }
+
+  public static final class prefIntake {
+    public static final SN_DoublePreference intakeSpeed = new SN_DoublePreference("intakeSpeed", 0.5);
```

Comment:
We may want to control the speed of the centering wheels and main roller independently as they are different motor types and different gearing.